packages/date-picker/src/DatePicker/DatePicker.scss

```diff
@@ -33,15 +33,15 @@ $placeholder-opacity: 0.7;
   border-radius: $border-solid-border-radius;
   padding: 0 $button-base-padding-horizontal;
 
-  &:focus:not([disabled]),
-  &:hover:focus:not([disabled]),
+  &:global(.focus-visible):not([disabled]),
+  &:hover:global(.focus-visible):not([disabled]),
   &:hover:not([disabled]) {
     background-color: $color-gray-200;
     border-color: $color-gray-600;
     color: $color-purple-800;
   }
 
-  &:focus:not([disabled]) {
+  &:global(.focus-visible):not([disabled]) {
```

Comment:
I believe you'll also want to do on top of this, so that the browser default outline doesn't show on click focus.

```
:focus {
  outline: none;
}
```

(This applies to all of the ones you've done here)

Reply:
Updated this :) thanks